kodilla-good-paterns/src/main/java/com/kodilla/good/patterns/challenges/Product.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.good.patterns.challenges;
+
+public class Product {
+    private String productName;
```

Comment:
final

Reply:
Nie wystarczy name? Przeciez wiadomo, ze to bedzie name produktu skoro klasa product